client/js/lounge.js

```diff
@@ -1,3 +1,7 @@
+var $ = require('jquery-browserify');
```

Comment:
Can you put all these variables inside the anonymous scope. We really wouldn't want to pollute the global namespace.